src/pretix/base/models/event.py

```diff
@@ -544,6 +546,59 @@ def get_users_with_permission(self, permission):
             Q(is_superuser=True) | Q(twp=True)
         )
 
+    @staticmethod
+    def has_paid_things(event):
+        item = apps.get_model('pretixbase', 'Item')
```

Comment:
You can just use a local import, i.e. ``from . import Item`` inside this method, you just can't have it in the file header to avoid the circular import. Using ``apps.get_model`` breaks tab completion in IDEs etc.